.circleci/config.yml

```diff
@@ -76,62 +92,54 @@ jobs:
                 exit 1
               fi
 
-      - checkout
-      - *post_checkout
-      - run:
-          name: Skip screenshotter if only documentation has been changed
-          command: |
-              if [[ $CHANGED ]]; then
-                echo "$CHANGED" | grep -qvE '^docs/|^LICENSE|\.md$' || circleci step halt
-              fi
-
-      - *restore_node_modules_cache
-      - *yarn_install
-      - save_cache:
-          key: *node_modules_cache_key
-          paths:
-            - node_modules
+      - checkout_repo
+      - skip_if_only_changed:
+          filter: '^docs/|^LICENSE|\.md$'
```

Comment:
Why not `filter: '^docs/|^static/|^website/|^LICENSE|\\.md$'` as above?

Reply:
@edemaine It's because we lint JS and CSS in `static` and `website`.